src/TTGamesExplorerRebirthUI/Forms/ImageForm.cs

```diff
@@ -177,7 +208,7 @@ private void TrackBar1_Scroll(object sender, EventArgs e)
             _zoomVal = trackBar1.Value;
 
             darkLabel1.Text = $"{_zoomVal}%";
-
+            pictureBox1.Image.Dispose(); // avoid memory leak
```

Comment:
```suggestion

            pictureBox1.Image.Dispose();

```

I'm not sure this is really needed.